### PR TITLE
chore: Replace fitInSuperview() - Part 3 IC-67

### PR DESCRIPTION
**Pull request by KaterinaWire**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/IC-67" title="IC-67" target="_blank">SubtaskIC-67</a>  'fitInSuperview(safely:with:exclude:activate:)' was deprecated in iOS 13.0: Use the anchors API instead
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Replace fitInSuperview() as it is deprecated on iOS 13. Part 3

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+Constraints.swift [hunk 1]
```diff
@@ -51,13 +55,20 @@ extension ConversationViewController {
...
+        NSLayoutConstraint.activate([
+            conversationBarController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            conversationBarController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            conversationBarController.view.topAnchor.constraint(equalTo: view.topAnchor)
+        ])
+        NSLayoutConstraint.activate([
+            contentViewController.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            contentViewController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            contentViewController.view.topAnchor.constraint(equalTo: view.topAnchor)
+        ])
```
Comment (agisilaos) [inline Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+Constraints.swift:67]: Same here

File: Wire-iOS/Sources/UserInterface/Services/ServiceView.swift [hunk 2]
```diff
@@ -66,9 +66,17 @@ final class ServiceDetailView: UIView {
...
+            serviceView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            serviceView.topAnchor.constraint(equalTo: topAnchor)
+        ])
 
-        descriptionTextView.fitInSuperview(exclude: [.top])
+        NSLayoutConstraint.activate([
+            descriptionTextView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            descriptionTextView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            descriptionTextView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
```
Comment (agisilaos) [inline Wire-iOS/Sources/UserInterface/Services/ServiceView.swift:79]: Same here

**Review by agisilaos — COMMENTED** [review]
Left some comments, let's take care of the SwiftLint warning as well.

File: Wire-iOS/Sources/UserInterface/Services/ServiceView.swift (hunk 2 above)
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Services/ServiceView.swift]: Done

File: Wire-iOS/Sources/UserInterface/Services/ServiceDetailViewController.swift [hunk 3]
```diff
@@ -153,15 +153,22 @@ final class ServiceDetailViewController: UIViewController {
...
+        NSLayoutConstraint.activate([
+            actionButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+            actionButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -16),
+            actionButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -(16 + UIScreen.safeArea.bottom))
+        ])
 
         NSLayoutConstraint.activate([
             detailView.topAnchor.constraint(equalTo: safeTopAnchor, constant: 16),
             actionButton.topAnchor.constraint(equalTo: detailView.bottomAnchor, constant: 16),
             actionButton.heightAnchor.constraint(equalToConstant: 48)
```
[the remark this answers is not in the document]
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/Services/ServiceDetailViewController.swift]: Done

File: Wire-iOS/Sources/UserInterface/ConversationList/UserNameTakeOverViewController.swift [hunk 4]
```diff
@@ -112,8 +112,15 @@ final class UserNameTakeOverViewController: UIViewController {
...
-        suggestedHandleLabel.fitInSuperview(exclude: [.top, .bottom])
+        NSLayoutConstraint.activate([
+            displayNameLabel.leadingAnchor.constraint(equalTo: topContainer.leadingAnchor),
+            displayNameLabel.trailingAnchor.constraint(equalTo: topContainer.trailingAnchor)
+        ])
+
+        NSLayoutConstraint.activate([
+            suggestedHandleLabel.leadingAnchor.constraint(equalTo: topContainer.leadingAnchor),
+            suggestedHandleLabel.trailingAnchor.constraint(equalTo: topContainer.trailingAnchor)
+        ])
```
[the remark this answers is not in the document]
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/ConversationList/UserNameTakeOverViewController.swift]: Done

File: Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIInviteActionBar.swift [hunk 5]
```diff
@@ -62,7 +62,9 @@ final class StartUIInviteActionBar: UIView {
     private func createConstraints() {
         inviteButton.translatesAutoresizingMaskIntoConstraints = false
 
-        bottomEdgeConstraint = inviteButton.fitInSuperview(with: EdgeInsets(top: padding, leading: padding * 2, bottom: padding + UIScreen.safeArea.bottom, trailing: padding * 2))[.bottom]
```
[the remark this answers is not in the document]
Comment (KaterinaWire) [reply Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIInviteActionBar.swift:65]: Yes. Fixed

**Review by agisilaos — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**